app/views/meditations/index.html.slim

```diff
@@ -1,99 +1,80 @@
 
-    .container
-      .top-left.sidetext Set Up your own meditation
-      h2.title Set Up your own Meditation
-      form
-        input type="hidden" name="goal_filters" method="get"
-        .select-meditation
-          .select-goals
-            .goals
-              a.active-goal.disable
-                span.active-goal-icon = inline_svg 'header/logo-mini.svg'
-                span.active-goal-header
-                  |I wish I feel
+section.custom-meditation
+  .container
+    .top-left.sidetext Set up your own meditation
+    h2.title Set up your own Meditation
+    form
+      input type="hidden" name="goal_filters" method="get"
+      .select-meditation
+        .select-goals
+          .goals
+            a.active-goal.disabled
+              span.active-goal-icon = inline_svg 'header/logo-mini.svg'
+              span.active-goal-header
+                | I want to feel
 
-                span.active-goal-text Choose feelings
+              span.active-goal-text Choose feelings
 
-              .dropdown
-                a.btn-close
-                h3.title Choose feelings
-                ul
-                  - @goal_filters.each_with_index do |goal_filter, index|
-                    li data-img="#{goal_filter.icon.url}"
-                      span
-                        = goal_filter.name
-          .duration
-            span.duration-title I have
-            .radio-wrap
-              /- @duration_filters.sample(3).each do |duration_filter|
-                /label.radio
-                  /input type="radio" name="radio" value="#{duration_filter.id}"
-                  /.radio-text
-                   / span #{duration_filter.minutes} min
-              label.radio
-                input type="radio" name="duration_filter" value="5"
-                  .radio-text
-                    span 5 min
-              label.radio
-                input type="radio" name="duration_filter" value="10"
-                  .radio-text
-                    span 10 min
+            .dropdown
+              a.btn-close
+              h3.title Choose feelings
+              ul
+                - @goal_filters.each_with_index do |goal_filter, index|
+                  li data-img="#{goal_filter.icon.url}"
+                    span
+                      = goal_filter.name
+        .duration
+          span.duration-title I have
+          .radio-wrap
+            - @duration_filters.first(3).each do |duration_filter|
               label.radio
-                input type="radio" name="duration_filter" value="15"
-                  .radio-text
-                    span 15 min
-
-        button.start-meditation type="submit" Start meditation
-        .mobile-img
-          img src=image_path('sections/try-meditation/own-meditate-mobile.png')
-
+                input type="radio" name="radio" value="#{duration_filter.id}"
+                .radio-text
+                  span #{duration_filter.minutes} min
```

Comment:
Changed this to use the first 3 durations instead of being absolutely hardcoded.